cvat/apps/engine/static/engine/js/shapeCreator.js

```diff
@@ -125,7 +125,7 @@ class ShapeCreatorModel extends Listener {
     }
 
     set defaultType(type) {
-        if (!['box', 'points', 'polygon', 'polyline'].includes(type)) {
+        if (!['box', 'box_by_4_points', 'points', 'polygon', 'polyline'].includes(type)) {
```

Comment:
There is incompatibilities with this code: https://github.com/opencv/cvat/blob/e062c2309ecc648afc2e7261bdf41224a7508dcf/cvat/apps/dextr_segmentation/static/dextr_segmentation/js/enginePlugin.js#L57

Reply:
Fixed this here: https://github.com/opencv/cvat/pull/1111/commits/211efd883a5678e3742bf04cf52e2319003eed42#diff-2abd9452793a719d412491d1bfa0d501R57